packages/edit-site/src/components/global-styles/screen-revisions/use-global-styles-revisions.js

```diff
@@ -75,7 +72,12 @@ export default function useGlobalStylesRevisions() {
 		}
 
 		// Adds an item for unsaved changes.
-		if ( isDirty && ! isEmpty( userConfig ) && currentUser ) {
+		if (
+			isDirty &&
+			userConfig &&
+			Object.keys( userConfig ).length > 0 &&
```

Comment:
`userConfig` is already used as a potentially nullish object, which is why we're adding the extra checks.